cmd/gardener-extension-shoot-cert-service/app/options.go

```diff
@@ -41,12 +42,16 @@ func NewOptions() *Options {
 			// These are default values.
 			LeaderElection:          true,
 			LeaderElectionID:        controllercmd.LeaderElectionNameID(ExtensionName),
-			LeaderElectionNamespace: os.Getenv(extension.EnvLeaderElectionNamespace),
+			LeaderElectionNamespace: os.Getenv(shared.EnvLeaderElectionNamespace),
 		},
-		controllerOptions: &controllercmd.ControllerOptions{
+		shootControllerOptions: &controllercmd.ControllerOptions{
 			// This is a default value.
 			MaxConcurrentReconciles: 5,
 		},
+		controlPlaneControllerOptions: &controllercmd.ControllerOptions{
+			// This is a default value.
+			MaxConcurrentReconciles: 2,
```

Comment:
I'm wondering if this shouldn't be `1` because for the garden and seed use-cases there is only one extension resource at a time.

Reply:
If the garden cluster is also used as seed cluster, you have two extension resources. So I think it makes sense to keep the `2`.